src/smartpeak/source/core/RawDataProcessor.cpp

```diff
@@ -75,7 +75,23 @@ namespace SmartPeak
 
   ParameterSet RawDataProcessor::getParameterSchema() const
   {
-    return ParameterSet();
+    std::map<std::string, std::vector<std::map<std::string, std::string>>> param_struct({
+    {"mzML", {
+      {
+        {"name", "format"},
+        {"type", "string"},
+        {"value", "Chromeleon"},
+        {"description", ""},
+        {"valid_strings", "['Chromeleon','XML']"}
+      },
+      {
+        {"name", "zero_baseline"},
+        {"type", "bool"},
+        {"value", "false"},
+        {"description", ""},
```

Comment:
For the description, we can add the following:

"Zeros the baseline of the chromatogram by adjusting all points so that the minimum point is 0."